press/press/doctype/deploy_candidate/deploy_notifications.py

```diff
@@ -249,7 +248,7 @@ def create_build_failed_notification(
 	details = get_details(dc, exc)
 	doc_dict = {
 		"doctype": "Press Notification",
-		"team": dc.team,
+		"team": dc.candidate.team,
```

Comment:
`dc.candidate` will be the name of candidate not object
we need use `get_value` to fetch team for that.

Reply:
dc.candidate is:


```python
@cached_property
	def candidate(self) -> DeployCandidate:
		return frappe.get_doc("Deploy Candidate", self.deploy_candidate)
```

Reply:
Oh missed